src/ex4/server/services/itemMangerValidator.js

```diff
@@ -0,0 +1,30 @@
+export class ItemManagerValidator {
+    constructor() { 
+        this.isNumberRegex = /^[0-9]+$/;
+        this.isListRegex = /^[0-9, ]+$/;
+    }
+    isNumber(value) {
+        return this.isNumberRegex.test(value);
+    }
+    isList(value) {
```

Comment:
what's the input value? by the name it's very clear what this function do, but when i look at the code i'm not sure there's a match

Reply:
what would you recommend to change the name of value to?